runtime/cfdumper/romdump.c

```diff
@@ -20,6 +20,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
  *******************************************************************************/
 
+#include "omrcomp.h"
```

Comment:
Expecting `omrcomp.h` to be included first is a fragile solution. I suggest that https://github.com/eclipse-omr/omr/pull/7711 should update `util/a2e/headers/ctype.h` to define `_AE_BIMODAL` before including the system header `<ctype.h>`.

Reply:
Oh that does sound much better, I will try that and hopefully it works functionally the same. Sounds like it should.